commands/create_command.py

```diff
@@ -0,0 +1,30 @@
+from Icommand import ICommand
+import os, json, sys
+parent_dir = os.path.abspath(os.path.join(os.path.dirname(__file__), os.pardir))
+sys.path.append(parent_dir)
+from SchemaKeys import SchemaKeys
+
+class CreateCommand(ICommand):
+    def __init__(self, schema):
+        self.__validate_schema(schema)
+        self.data = json.load(open(schema, "r"))
+
+    def __validate_schema(self, schema):
+        if (schema == None or not os.path.exists(os.path.join(parent_dir, schema))):
+            raise Exception("FileNotFound")
+        data = json.load(open(schema, "r"))
+        if not SchemaKeys.DATABASE in data:
+            raise Exception("DatabaseNameIsMissing")
+        
+    def execute(self):    
```

Comment:
any execute function should return a status or status and message. 
I suggest to create an object call 
```
class CommandOutput:
__init__(status, message)

```

Reply:
Is this different from the enum class you showed us?

Reply:
You have two solution 
Solution #1 
CommandOutput which is a class containing two variables  status (enum) and message. The UI will use this message 

solution #2
Return a status only (enum) and the UI. will have a map between message and status.

the second solution is better if you need to translate the test to different languages. But in our database case, we don’t need this, so u can use the first solution.

Reply:
You have third solution which is returning and exception.

we can wrap the exception to output outside the command

Reply:
Any update on this ?

Reply:
working on it.